src/components/TimeLine/time-line.js

```diff
@@ -117,4 +122,51 @@ const processData = (data, colors) => {
   return dataSplitYears;
 };
 
+const processKtas = (ktas) => {
+
+  if (!ktas || ktas === []) return [];
+
+  let ktasSplitYears = [];
```

Comment:
ktasSplitYears ? 

is it a list of ktas by year ? 

ktasByYear or ktasYearlyBuckets  (or something better, these are pretty terrible as well :D)